app/models/user.rb

```diff
@@ -26,6 +26,7 @@ class User < ApplicationRecord
     attachable.variant :mini, resize_to_limit: [40, 40]
   end
 
+  before_update :add_verification_credits, if: :verified_at_changed?
```

Comment:
why before_update is used and not any other callback?

Reply:
So that I can use verified at changed because after the obj it saved it will be changed to nil and i will only be used when user updates he verified status by pressing the email_confirm_link